test/unit/org/apache/cassandra/nodes/virtual/LegacySystemKeyspaceToNodesTest.java

```diff
@@ -163,8 +162,8 @@ public void verifyLocalOnlyUpgrade() throws Exception
             removeTruncated(SystemKeyspace.BATCHES);
 
             umappedViews(() -> {
-                assertEquals(1, executeInternal("SELECT * FROM system.local").size());
-                assertTrue(executeInternal("SELECT * FROM system.peers").isEmpty());
+                assertEquals(1, QueryProcessor.executeInternal("SELECT * FROM system.local").size());
```

Comment:
Just preserving the original call to `QueryProcessor.executeInternal`, which has been shaded by `CQLTester#executeInternal`

Reply:
I am not sure why only for this test but not for others where we use `CQLTester#executeInternal`?

Reply:
This test originally called `QueryProcessor.executeInternal`, none of the `CQLTester` execute flavours. Since the test extends `CQLTester`, the new `CQLTester#executeInternal` masks `QueryProcessor.executeInternal`, complaining about the signature of the method. This is simply restoring the original call.

Reply:
Ohhh, it seems I saw it wrong that the original call was pointing to `CQLTester#executeInternal`. Sorry about that. You are right